src/interfaces/project.ts

```diff
@@ -24,7 +24,7 @@ export interface IProjectHealthReport extends IProject {
     staleCount: number;
     potentiallyStaleCount: number;
     activeCount: number;
-    updatedAt: Date;
+    updatedAt: string;
```

Comment:
is it really a type of string ?

Reply:
Yep! This type describes what we get from the backend.